packages/veui/test/unit/specs/mixins/activatable.spec.js

```diff
@@ -0,0 +1,55 @@
+import Vue from 'vue'
+import { createLocalVue, shallowMount } from '@vue/test-utils'
+import activatable from '@/mixins/activatable'
+
+const noop = function () {}
+
+const localVue = createLocalVue()
+
+const ActivatableComponent = {
+  template: '<div>Activatable Component</div>',
+  methods: {
+    activate: noop
+  },
+  mixins: [activatable]
+}
+
+const InActivatableComponent = {
+  template: '<div>InActivatable Component</div>',
+  methods: {
+    inactivate: noop
+  },
+  mixins: [activatable]
+}
+
+describe('mixins/activatable', () => {
+  let msg
+  beforeEach(() => {
+    Vue.config.warnHandler = message => {
+      msg = message
+    }
+  })
+
+  afterEach(() => {
+    Vue.config.warnHandler = undefined
+    msg = undefined
+  })
+
+  it('should has `activate` method', () => {
+    const { vm } = shallowMount(ActivatableComponent, {
+      localVue
+    })
+
+    expect(vm.activate).to.be.a('function')
+    expect(msg).to.be.undefined
```

Comment:
还是不要这么写了，为了这个还要再引 ESLint 插件……

```suggestion
    expect(msg).to.equal(undefined)
```

Reply:
好